test_rclcpp/CMakeLists.txt

```diff
@@ -135,6 +137,46 @@ if(AMENT_ENABLE_TESTING)
         "rclcpp")
     endif()
 
+    if(TARGET local_parameters_cpp__${middleware_impl})
+      ament_add_gtest(local_parameters_cpp__${middleware_impl} "test/local_parameters.cpp")
```

Comment:
What's the solution here? Remove the main function from `local_parameters.cpp` or not add it with `ament_add_gtest` ?